cfme/intelligence/chargeback/rates.py

```diff
@@ -224,19 +207,65 @@ def delete(self, cancel=False):
         """
         view = navigate_to(self, 'Details', wait_for_view=True)
         view.toolbar.configuration.item_select('Remove from the VMDB', handle_alert=(not cancel))
+        view = self.create_view(navigator.get_class(self.parent, 'All').VIEW)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+
+
+@attr.s
+class ComputeRateCollection(BaseCollection):
+    ENTITY = ComputeRate
+    RATE_TYPE = 'Compute'
+
+    def create(self, description=None, currency=None, fields=None):
```

Comment:
Question: can the rate be created without any parameters?

Reply:
@LandoCalrizzian ComputeRate class had them as optional parameters in the `__init__`, please double check this against the MIQ page that even the description is optional.  Add comments if there is some exception or substitution made for the model's attributes.  docblocks for these classes/methods are required as well, good place to put any such comments.

Reply:
@rlbabyuk Technically no it can't be created. The original implementation of create() would treat the lack of args as a way to cancel the creation using the view.fill_with(no_change=view.cancel_button). Updated to make the 'description' arg required.